docs/source/logging/logging.md

```diff
@@ -1,54 +1,125 @@
+To customise logging, specify the path of your logging configuration file by setting the environment variable `KEDRO_LOGGING_CONFIG` to override the default, which is Kedro's `default_logging.yml`. We recommend you put `logging.yml` inside the `conf` folder. For example, you can set `KEDRO_LOGGING_CONFIG` as follows:
 
 ```bash
-export KEDRO_LOGGING_CONFIG=/path/to/logging.yml
+export KEDRO_LOGGING_CONFIG=<project_root>/conf/logging.yml
 ```
 
 After setting the environment variable, any subsequent Kedro commands will use the logging configuration file at the specified path.
 
 ```{note}
 If the `KEDRO_LOGGING_CONFIG` environment variable is not set, Kedro will default to using the logging configuration file at the project's default location of  Kedro's `default_logging.yml`.
 ```
-### Disable file-based logging
 
-You might sometimes need to disable file-based logging, e.g. if you are running Kedro on a read-only file system such as [Databricks Repos](https://docs.databricks.com/repos/index.html). The simplest way to do this is to delete your `conf/base/logging.yml` file. With no project-side logging configuration specified, Kedro uses the default framework-side logging configuration, which does not include any file-based handlers.
+### Show DEBUG level messages
+To see `DEBUG` level messages, change the level of logging in `logging.yml`:
 
-Alternatively, if you would like to keep other configuration in `conf/base/logging.yml` and just disable file-based logging, then you can remove the file-based handlers from the `root` logger as follows:
-```diff
- root:
--  handlers: [console, info_file_handler]
-+  handlers: [console]
+```yaml
+loggers:
+  kedro:
+    level: INFO
+
+  your_python_pacakge:
+    level: DEBUG  # Change this to DEBUG
+```
+
+By changing the level value to `DEBUG` for the desired logger (e.g., <your_python_package>), you will start seeing `DEBUG` level messages in the log output.
+
+# Advanced logging
+In addition to the `rich` handler defined in Kedro's framework, we provide a `logging.yml` template.
+
+<details>
+<summary><b>Click to expand the `logging.yml` template</b></summary>
+
+```yaml
+version: 1
+
+disable_existing_loggers: False
+
+formatters:
+  simple:
+    format: "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+
+handlers:
+  console:
+    class: logging.StreamHandler
+    level: INFO
+    formatter: simple
+    stream: ext://sys.stdout
+
+  info_file_handler:
+    class: logging.handlers.RotatingFileHandler
+    level: INFO
+    formatter: simple
+    filename: info.log
+    maxBytes: 10485760 # 10MB
+    backupCount: 20
+    encoding: utf8
+    delay: True
+
+  rich:
+    class: kedro.logging.RichHandler
+    rich_tracebacks: True
+    # Advance options for customisation.
+    # See https://docs.kedro.org/en/stable/logging/logging.html#project-side-logging-configuration
+    # tracebacks_show_locals: False
+
+loggers:
+  kedro:
+    level: INFO
+
+  {{ cookiecutter.python_package }}:
+    level: INFO
```

Comment:
```suggestion
```
We should remove this because it's confusing and not actually needed thanks to #2520.